examples/nlp/punctuation_capitalization.py

```diff
@@ -168,7 +182,7 @@ def create_pipeline(
 ):
     global punct_classifier, punct_loss, capit_classifier, capit_loss, task_loss
```

Comment:
ok, what is happening here? why those class objects are global??

Reply:
would be addressed in next stage.